### PR TITLE
[DSS-465]: chore() - housekeeping: update components to use tokens

### PR DESCRIPTION
**Pull request by pixelflips**
# Description
The following components have some hardcoded values that should be updated to use tokens:

- Button
- Link
- Tabs
- Tooltip
- Divider (has typo in offset values)
- Copy Text (has some left over comments)

These should be updated to use token values where applicable.

Fixes [DSS-465](https://kajabi.atlassian.net/browse/DSS-468)

## Type of change

- [x] Chore
- [x] This change requires a documentation update

# How Has This Been Tested?

Navigate to the components outlined above and verify there are no visual changes.

- [ ] unit tests
- [ ] e2e tests
- [ ] accessibility tests
- [x] tested manually
- [ ] other:

**Test Configuration**:

- Pine versions:
- OS:
- Browsers:
- Screen readers:
- Misc:

# Checklist:

If not applicable, leave options unchecked.

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes
- [ ] Design has QA'ed and approved this PR

[DSS-465]: https://kajabi.atlassian.net/browse/DSS-465?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ju-Skinner — APPROVED** [review]
:shipit:

File: libs/core/src/components/pds-tabs/pds-tab/pds-tab.scss [hunk 1]
```diff
@@ -1,4 +1,24 @@
...
+
+  --font-size-body: var(--pine-font-size-body);
+  --font-size-sm: var(--pine-font-size-body-sm);
+  --font-size-xs: var(--pine-font-size-body-xs);
+  --font-weight: var(--pine-font-weight-medium);
+  --line-height-md: var(--pine-line-height-md);
+  --line-height-sm: var(--pine-line-height-sm);
+  --line-height-xs: var(--pine-line-height-xs);
+
+  --outline: 4px solid var(--pine-color-primary-200);
```
Comment (ju-Skinner) [inline libs/core/src/components/pds-tabs/pds-tab/pds-tab.scss:20]: Is there a reason you didn't use `var(--outline)` here like you did below?

**Review by ju-Skinner — COMMENTED** [review]
Found something on my second pass. See comment below

File: libs/core/src/components/pds-tabs/pds-tab/pds-tab.scss (hunk 1 above)
Comment (pixelflips) [reply libs/core/src/components/pds-tabs/pds-tab/pds-tab.scss:20]: @ju-Skinner Not sure I understand the question here. Can you clarify?

**Review by anechol — APPROVED** [review]
This is extremely helpful, so thank you for taking this work on.

File: libs/core/src/components/pds-tabs/pds-tab/pds-tab.scss (hunk 1 above)
Comment (ju-Skinner) [reply libs/core/src/components/pds-tabs/pds-tab/pds-tab.scss:20]: NVM... 🤦🏾 It was early in the morning when I looked at this again. Ignore. This was the definition for the variable.

**Pull request merged**